requirements.txt

```diff
@@ -21,12 +21,12 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 #####################################################################################
-google/protobuf@v3.11.0 -DCMAKE_POSITION_INDEPENDENT_CODE=On -X subdir -Dprotobuf_BUILD_TESTS=Off
+google/protobuf@v3.20.2 -DCMAKE_POSITION_INDEPENDENT_CODE=On -X subdir -Dprotobuf_BUILD_TESTS=Off
```

Comment:
I wonder if `.pb` files or `.onnx` needs to be re-generated. 
Did all tests pass ?

Reply:
that error is from CK. 
You need to rebuild your dependencies using `rbuild develop` or `rbuild prepare`.

Reply:
I tried `rbuild develop` but I still saw the same error. The I tried `rbuild prepare` with the command ` rbuild prepare --deps-dir ../AMDMIGraphX/`. But I still see the same error. I am doing this in the build directory in a docker container. Do you know what I might be missing? :)

Reply:
Never mind, I think I got it fixed. Doing `rbuild prepare -d depend` and using `-DCMAKE_PREFIX_PATH=depend` seems to have done the trick.

Reply:
> I wonder if `.pb` files or `.onnx` needs to be re-generated. Did all tests pass ?

@umangyadav All tests except the test_verify ones I mentioned in the other message pass.